packages/amplify-opensearch-simulator/package.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "@aws-amplify/amplify-opensearch-simulator",
+    "version": "1.0.0",
+    "description": "Opensearch local simulator",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/aws-amplify/amplify-cli.git",
+        "directory": "packages/amplify-opensearch-simulator"
+    },
+    "author": "Amazon Web Services",
+    "license": "Apache-2.0",
+    "main": "index.js",
+    "keywords": [
+        "graphql",
+        "appsync",
+        "aws"
+    ],
+    "scripts": {
+        "test": "jest"
+    },
+    "dependencies": {
+        "amplify-cli-core": "3.3.0",
+        "aws-sdk": "^2.1113.0",
+        "detect-port": "^1.3.0",
+        "execa": "^5.1.1",
+        "fs-extra": "^8.1.0",
+        "logdown": "^3.3.0",
+        "openpgp": "^4.10.10",
```

Comment:
This package is nearly 22mb in size unpacked according to [npm](https://www.npmjs.com/package/openpgp). Is it absolutely necessary to add this?

Reply:
The [recommended way](https://opensearch.org/verify-signatures.html) to verify the authenticity of downloaded build file is using the public PGP key that is provided. This is the only reliable helper package for us to take advantage of.